### PR TITLE
couple of typos; style edits

### PR DESCRIPTION
**Pull request by jspaaks**
Note this was branched of off ``paper_review`` branch.

This PR corrects some typos and such.

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -57,33 +57,33 @@ bibliography: paper.bib
 
 # Summary
 
-Mass spectrometry data is at the heart of numerable applications in the biomedical and life sciences.
-With growing use of high throughput techniques researchers need to analyse larger and more complex datasets. In particular through joint effort in the research community, fragmentation mass spectrometry datasets are growing in size and number.
+Mass spectrometry data is at the heart of numerous applications in the biomedical and life sciences.
+With growing use of high-throughput techniques, researchers need to analyze larger and more complex datasets. In particular through joint effort in the research community, fragmentation mass spectrometry datasets are growing in size and number.
 Platforms such as MassBank [@horai_massbank_2010], GNPS [@Wang2016] or MetaboLights [@haug_metabolights_2020] serve as an open-access hub for sharing of raw, processed, or annotated fragmentation mass spectrometry data (MS/MS).
```
Comment (jspaaks) [inline paper/paper.md]: "mass spectrometry" and "(MS/MS)" are not equivalent terms but the current formulation does suggest that.

File: paper/paper.md [hunk 2]
```diff
@@ -57,33 +57,33 @@ bibliography: paper.bib
...
 Without suitable tools, however, exploitation of such datasets remains overly challenging. 
-In particular, large collected datasets contain data aquired using different instruments and measurement conditions, and can further contain a significant fraction of inconsistent, wrongly labeled, or incorrect metadata (annotations).
+In particular, large collected datasets contain data acquired using different instruments and measurement conditions, and can further contain a significant fraction of inconsistent, wrongly labeled, or incorrect metadata (annotations).
 
-``Matchms`` is an open-access Python package to import, process, clean, and compare mass spectrometry data (MS/MS) (see \autoref{fig:flowchart}).
+``matchms`` is an open-access Python package to import, process, clean, and compare mass spectrometry data (MS/MS) (see \autoref{fig:flowchart}).
```
Comment (jspaaks) [inline paper/paper.md:66]: same here

**Comment by jspaaks** [comment]
Also the PNG includes a bullet point "Normalise" which should be "Normalize" :us:

**Comment by jspaaks** [comment]
Also, some bullet points in the 1st PNG mention "m/z" whereas the 2nd PNG uses "M/z". I suggest using "M/z" throughout.

File: paper/paper.md (hunk 1 above)
Comment (florian-huber) [reply paper/paper.md]: Yes, MS/MS stands for tandem mass spectrometry I believe. But "fragmentation mass spectrometry" and MS/MS are also often used as equivalents although that might technically not be correct... but I am not the expert here (I'll check).

File: paper/paper.md (hunk 2 above)
Comment (florian-huber) [reply paper/paper.md:66]: I'll add those changes, thanks!

**Comment by florian-huber** [comment]
Thanks @jspaaks! I will add the changes.
I will switch to "m/z" consistently tough since that is what I see most often.

File: paper/paper.md (hunk 1 above)
Comment (florian-huber) [reply paper/paper.md]: I removed the (MS/MS) here since it is not necessary anyway.

**Review by florian-huber — APPROVED** [review]
Thanks for the corrections!

**Pull request merged**